markhor_estop/src/estop.cpp

```diff
@@ -0,0 +1,51 @@
+#include "ros/ros.h"
+#include "std_srvs/Trigger.h"
+#include "JetsonXavierGPIO/jetsonGPIO.c"
```

Comment:
I know this looks really weird and probably wrong. But I suspect that the CMake is not correctly setup to integrate the lib into the executable. So this is the present workaround. I'll create an issue for this.